lib/Command/ClassifyImages.php

```diff
@@ -90,9 +90,6 @@ protected function execute(InputInterface $input, OutputInterface $output): int
                 $this->imagenet->classify($images);
 
                 $faces = $this->referenceFacesFinder->findReferenceFacesForUser($user);
-                if (count($faces) === 0) {
```

Comment:
I'm happy with putting these checks into the respective `classify` methods (above as well).

Instead of putting the enabled checks into the finder logic, I'd say, let's put them explicitly into the command and background job classes. What do you think?

Reply:
Yeah I didn't want to duplicate some code but that would certainly make more sense to have this check in the Service and the Command

Reply:
Let's duplicate the check for now. Eventually, we'll have to factor out the commonalities of the command and the background job into a common class anyway.